packages/renderer/src/lib/kube/KubernetesDashboard.svelte

```diff
@@ -0,0 +1,90 @@
+}
+</script>
+
+<NavPage searchEnabled={false}  title="Dashboard">
+  <svelte:fragment slot="bottom-additional-actions">
+    <div class="flex grow justify-end">
+      <KubernetesCurrentContextConnectionBadge />
+    </div>
+  </svelte:fragment>
+
+  <div class="flex min-w-full h-full" slot="content">
+    {#if noContexts}
+        <KubernetesEmptyPage />
+    {:else}
+        <div class="flex flex-col space-y-4 min-w-full overflow-y-auto">
+            <div class="flex flex-col gap-4 p-5 pb-2">
+                <span>Here you can manage and interact with Kubernetes clusters with features like connecting to clusters, and
+                viewing workloads like deployments and services.</span>
+
+                <span>Get up and running by clicking one of the menu items!</span>
+
+                <Link class="place-self-start" on:click={openKubernetesDocumentation}>Kubernetes documentation</Link>
+            </div>
+
+            <div class="flex flex-col gap-4 bg-[var(--pd-content-card-bg)] grow p-5">
+                {#if currentContextName}
+                    <div class="text-xl pt-2">Metrics</div>
+                    <div class="grid grid-cols-4 gap-4">
+                        <KubernetesDashboardResourceCard type='Nodes' Icon={NodeIcon} count={nodeCount} link='/kubernetes/nodes'/>
+                        <KubernetesDashboardResourceCard type='Deployments' Icon={DeploymentIcon} count={deploymentCount} link='/kubernetes/deployments'/>
+                        <KubernetesDashboardResourceCard type='Services' Icon={ServiceIcon} count={serviceCount} link='/kubernetes/services'/>
+                        <KubernetesDashboardResourceCard type='Ingresses & Routes' Icon={IngressRouteIcon} count={ingressRouteCount} link='/kubernetes/ingressesRoutes'/>
+                        <KubernetesDashboardResourceCard type='Persistent Volume Claims' Icon={PvcIcon} count={pvcCount} link='/kubernetes/persistentvolumeclaims'/>
+                        <KubernetesDashboardResourceCard type='ConfigMaps & Secrets' Icon={ConfigMapSecretIcon} count={configMapSecretCount} link='/kubernetes/configmapsSecrets'/>
+                    </div>
+                {/if}
+
+                <div class="text-xl pt-2">Explore articles and blog posts</div>
+                <div class="grid grid-cols-3 gap-4">
+                    <KubernetesDashboardGuideCard title='Deploy and test Kubernetes containers using Podman Desktop' link='https://developers.redhat.com/articles/2023/06/09/deploy-and-test-kubernetes-containers-using-podman-desktop'/>
+                    <KubernetesDashboardGuideCard title='Working with Kubernetes in Podman Desktop' link='https://developers.redhat.com/articles/2023/11/06/working-kubernetes-podman-desktop'/>
+                    <KubernetesDashboardGuideCard title='Share your local podman images with the Kubernetes cluster' link='https://podman-desktop.io/blog/sharing-podman-images-with-kubernetes-cluster'/>
```

Comment:
todo: create a follow up issue as it needs to be moved out to the backend, it should not be harcoded like that
it needs to be like for guided tour, in a json file

Reply:
I was assuming this should be part of the existing follow up, #9265. I've added comments there to be explicit and note all of the current gaps I'm aware of.